roles/matrix-prometheus/templates/systemd/matrix-prometheus.service.j2

```diff
@@ -31,7 +31,8 @@ ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-prometheus \
 			{% for arg in matrix_prometheus_container_extra_arguments %}
 			{{ arg }} \
 			{% endfor %}
-			{{ matrix_prometheus_docker_image }}
+			{{ matrix_prometheus_docker_image }} \
+			{{ matrix_prometheus_process_extra_arguments|join(' ') }}
```

Comment:
(Untested) If `matrix_prometheus_process_extra_arguments` is `[]`, then this may result in an empty line.. Which I'm not sure if it will cause a problem or not.. but it will certainly look weird.

It may be better to put these on the same line, just in case.

```suggestion
			{{ matrix_prometheus_docker_image }} {{ matrix_prometheus_process_extra_arguments|join(' ') }}
```

Reply:
the idea was to always have defaults in the extra_arguments var, but yes, that's a valid point!